tools/flang2/flang2exe/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ set_source_files_properties(${UTILS_SYMTAB_BIN_DIR}/symtabdf.cpp PROPERTIES GENE
 
 set(SOURCES
   ${ARCH_DEP_FILES}
+  asprintf.c
```

Comment:
Files in flang2 were all converted to cpp files sometime back. Is it possible to have this file also as a cpp file?